js/apps/system/_admin/aardvark/APP/frontend/js/views/installGitHubServiceView.js

```diff
@@ -73,35 +73,36 @@
 
     installFoxxFromGithub: function () {
       if (window.modalView.modalTestAll()) {
-        var url, version, mount, flag, isLegacy;
+        var mount, info, options, url, version;
         if (this._upgrade) {
           mount = window.App.replaceAppData.mount;
-          flag = arangoHelper.getFoxxFlag();
         } else {
           mount = window.arangoHelper.escapeHtml($('#new-app-mount').val());
           if (mount.charAt(0) !== '/') {
             mount = '/' + mount;
           }
         }
+
         url = window.arangoHelper.escapeHtml($('#repository').val());
         version = window.arangoHelper.escapeHtml($('#tag').val());
-
         if (version === '') {
           version = 'master';
         }
-        var info = {
-          url: window.arangoHelper.escapeHtml($('#repository').val()),
-          version: window.arangoHelper.escapeHtml($('#tag').val())
```

Comment:
This is literally the same code as a few lines earlier except it ignores the default for "master". This is likely a bug.